core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token_fr.properties

```diff
@@ -0,0 +1,13 @@
+authenticate-security-token.getting.started=D\u00e9marrage
```

Comment:
NIT: as .properties files are by definition encoded into ISO-8859-1, I think this would be easier to just write 'Démarrage' instead for readability reasons. 
I fixed something in a translation file some times ago because some character was actually written twice, but that would only be really visible rendered in the UI. (OK, with an editor that would support rendering it, that might be less of an issue, but still in a PR it's more complex to review).